server/src/main/java/org/apache/druid/segment/loading/SegmentLoaderConfig.java

```diff
@@ -52,6 +53,10 @@
   @JsonProperty("numBootstrapThreads")
   private Integer numBootstrapThreads = null;
 
+  @JsonProperty("locationSelectorStrategy")
+  private StorageLocationSelectorStrategy locationSelectorStrategy =
```

Comment:
Please fix `toString()` method of this class.

Reply:
done